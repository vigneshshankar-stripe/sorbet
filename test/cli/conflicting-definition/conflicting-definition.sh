#!/bin/bash
set -eu

main/sorbet --silence-dev-message --stop-after=namer \
  test/cli/conflicting-definition/{a,b}.rb 2>&1
