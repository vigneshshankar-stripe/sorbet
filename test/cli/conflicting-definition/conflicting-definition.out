test/cli/conflicting-definition/b.rb:1: `Foo` has behavior defined in multiple files https://srb.help/4018
     1 |module Foo
        ^^^^^^^^^^
    test/cli/conflicting-definition/a.rb:1: Previous definition
     1 |module Foo
        ^^^^^^^^^^
Errors: 1
